.gitlab-ci.yml

```diff
@@ -0,0 +1,87 @@
+
+# /************************************************************************
+ #  File: .gitlab-ci.yml
+ #  Author: mdr0id
+ #  Date: 9/10/2018
+ #  Description:  Used to setup runners/jobs for librustzcash
+ #  Usage: Commit source and the pipeline will trigger the according jobs.
+ #
+ #  Known bugs/missing features:
+ #
+ # ************************************************************************/
+  
+stages:
+  - build
+  - test
+  - deploy
```

Comment:
Are these stages sequential? I'm confused by the fact that the tasks below with the same "line" have different names with no obvious linking, so it's not clear how subsequent stages benefit from earlier ones.

Reply:
These stages are sequential, per some platform image. The blocks below are like method definitions in a .cpp file. They can be in any order, per definition, but this block tells CI to run stages in this order.